README.md

```diff
@@ -1,36 +1,520 @@
-# Terraform Cloud Operator for Kubernetes (terraform-k8s)
+# Terraform Cloud Operator for Kubernetes (TFCOK)
 
-> This experimental repository contains software which is still being developed
-> and in the alpha testing stage. It is not ready for production use.
+> This repository contains experimental software which is still under development and in the alpha testing stage. It is not ready for production use.
 
-The `terraform-k8s` binary includes first-class integrations between Terraform Cloud/Enterprise
-and Kubernetes. Currently, this project only includes the Terraform Cloud
-Operator, which synchronizes a Kubernetes Workspace (Custom Resource) to a
-Terraform Cloud Workspace. You can read more about this project and its potential use cases on our [blog](https://www.hashicorp.com/blog/creating-workspaces-with-the-hashicorp-terraform-operator-for-kubernetes/).
-We are actively considering other possible use cases to add to this project outside of the operator, and welcome your feedback. 
+The Terraform Cloud Operator for Kubernetes provides first-class integration between [Kubernetes](https://kubernetes.io/) and [Terraform Cloud](https://www.hashicorp.com/products/terraform/editions/cloud/) by extending the Kubernetes control plane to enable lifecycle management of cloud and on-prem infrastructure through [Kubernetes manifests](https://kubernetes.io/docs/concepts/cluster-administration/manage-deployment/) by using kubectl, Terraform, Gitops tools, or any other tool that allows you to manage Kubernetes [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/). 
```

Comment:
How about breaking this sentence at `Kubernetes manifests` and then making a new sentence. You can manage the manifests by using kubectl...etc

Reply:
Let me know what you think about the change.